### PR TITLE
adds not_human and gondolas

### PR DESCRIPTION
**Pull request by MadmanMartian**
moves martian under this 'not human' object type

adds gondola. Walking, silent media receivers. (Unable to test in local, will likely need testing with a pomf media server)

:cl:
* rscadd: Peace, tranquility, and gondola.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
I think the type should have a better name. Maybe something like `/complex`?
For instance, monkeys aren't human and yet they would also not be `/not_human`.

**Comment by ihadtoregisterforthis** [comment]
The side sprites are very lacking

File: code/modules/media/broadcast/transmitter.dm [hunk 1]
```diff
@@ -34,6 +34,9 @@ var/global/media_transmitters=list()
 			if(R.media_crypto == media_crypto)
 				R.receive_broadcast(media_url,media_start_time)
 				//testing("[src]: Sending music to [R]")
+		for(var/mob/living/carbon/not_human/gondola/G in media_receivers[freq])
+			if(G.media_crypto == media_crypto)
+				G.receive_broadcast(media_url,media_start_time)
```
Comment (Kurfursten) [inline code/modules/media/broadcast/transmitter.dm]: iirc media receivers do not play well with crossing room boundaries (e.g.: boombox) and simply won't update their playing area unless turned off and on

**Comment by Jubisloviu** [comment]
now we just need a gondola playlist of comfy songs

**Comment by SonixApache** [comment]
please let this not be a april fools joke

**Comment by Shadowmech88** [comment]
Also, I think ideally humans should be children of `/complex` as well, since I don't think there's anything such mobs should be able to do that a human *shouldn't* be able to do.
However, reparenting `/human` would be quite an involved task and may cause issues that would need to be fixed.

Actually, the *real* ideal solution would be to do away with `/complex` and just make martians and gondolas species of humans, but that would involve reworking organs and limbs and such since human code was not built to handle species that, for example, don't have torsos.

**Comment by Kurfursten** [comment]
You solved the issue with media receivers crossing boundaries?

**Comment by Probe1** [comment]
@SonixApache It is but in keeping with tradition it will mysteriously not be removed

**Comment by Probe1** [comment]
90% chance I have to revert this when the server explodes.

**Comment by Probe1** [comment]
God bless it didn't explode.

**Pull request merged**